src/translations/main_tr.ts

```diff
@@ -0,0 +1,822 @@
+    <message>
+        <source>Show the Thunderbird window.</source>
+        <translation>Thunderbird penceresini göster.</translation>
+    </message>
+    <message>
+        <source>Hide the Thunderbird window.</source>
+        <translation>Thunderbird penceresini gizleyin.</translation>
+    </message>
+    <message>
+        <source>Write log to a file.</source>
+        <translation>Log&apos;u bir dosyaya yazın.</translation>
+    </message>
+    <message>
+        <source>FILE</source>
+        <translation>DOSYA</translation>
+    </message>
+</context>
+<context>
+    <name>DialogAddEditNewEmail</name>
+    <message>
+        <source>&quot;New Email&quot; Entry</source>
+        <translation>&quot;Yeni E-posta&quot; Girişi</translation>
+    </message>
+    <message>
+        <source>Name:</source>
+        <translation>İsim:</translation>
+    </message>
+    <message>
+        <source>(Optional) Prefilled fields</source>
+        <translation>(İsteğe bağlı) Önceden doldurulmuş alanlar</translation>
+    </message>
+    <message>
+        <source>Email subject:</source>
+        <translation>E-posta konu:</translation>
+    </message>
+    <message>
+        <source>Email message text:</source>
+        <translation>E-posta mesajı metni:</translation>
+    </message>
+    <message>
+        <source>Email recipient:</source>
+        <translation>E-posta alıcısı:</translation>
+    </message>
+    <message>
+        <source>No name specified</source>
+        <translation>İsim belirtilmedi</translation>
+    </message>
+    <message>
+        <source>The name cannot be empty.</source>
+        <translation>İsim boş olamaz.</translation>
+    </message>
+</context>
+<context>
+    <name>DialogLogOutput</name>
+    <message>
+        <source>Debug log viewer</source>
+        <translation>Hata ayıklama günlük görüntüleyicisi</translation>
+    </message>
+</context>
+<context>
+    <name>DialogSettings</name>
+    <message>
+        <source>Birdtray Settings</source>
+        <translation>Birdtray Ayarları</translation>
+    </message>
+    <message>
+        <source>General</source>
+        <translation>General</translation>
+    </message>
+    <message>
+        <source>New Mail Notification</source>
+        <translation>Yeni Posta Bildirimi</translation>
+    </message>
+    <message>
+        <source>Show unread count</source>
+        <translation>Okunmamış sayıyı göster</translation>
+    </message>
+    <message>
+        <source>Different icon when unread:</source>
+        <translation>Okunmadığında farklı simge:</translation>
+    </message>
+    <message>
+        <source>Off</source>
+        <translation>Kapalı</translation>
+    </message>
+    <message>
+        <source>Fastest</source>
+        <translation>En hızlı</translation>
+    </message>
+    <message>
+        <source>Icon (Ctrl-click to reset):</source>
+        <translation>Simge (sıfırlamak için Ctrl tuşunu basılı tutarak tıklayın):</translation>
```

Comment:
You might want to try to find something shorter here, because this takes up a lot of space on the first settings page:
![Settings page with a lot of unused space because of this line](https://user-images.githubusercontent.com/6966049/94533736-db472480-023f-11eb-84c4-91417242d737.png)

Reply:
Alternatively, you could use a second line:
```suggestion
        <translation>Simge (sıfırlamak için Ctrl
tuşunu basılı tutarak tıklayın):</translation>
```
![Settings page with a newline in the text](https://user-images.githubusercontent.com/6966049/94534666-f1a1b000-0240-11eb-8e46-846435993b6b.png)